mmv1/products/cloudrun/api.yaml

```diff
@@ -616,6 +616,84 @@ objects:
                     required: true
                     description: |-
                       This must match the Name of a Volume.
+            - !ruby/object:Api::Type::NestedObject
+              name: startupProbe
+              min_version: beta
+              description: |-
+                Startup probe of application within the container.
+                All other probes are disabled if a startup probe is provided, until it
+                succeeds. Container will not be added to service endpoints if the probe fails.
+              properties:
+                - !ruby/object:Api::Type::Integer
+                  name: initialDelaySeconds 
+                  description: |-
+                    Number of seconds after the container has started before the probe is
+                    initiated.
+                    Defaults to 0 seconds. Minimum value is 0. Maximum value is 240.
+                  default_value: 0
+                - !ruby/object:Api::Type::Integer
+                  name: timeoutSeconds 
+                  description: |-
+                    Number of seconds after which the probe times out.
+                    Defaults to 1 second. Minimum value is 1. Maximum value is 3600.
+                    Must be smaller than periodSeconds.
+                  default_value: 1
+                - !ruby/object:Api::Type::Integer
+                  name: periodSeconds 
+                  description: |-
+                    How often (in seconds) to perform the probe.
+                    Default to 10 seconds. Minimum value is 1. Maximum value is 240.
+                  default_value: 10
+                - !ruby/object:Api::Type::Integer
+                  name: failureThreshold  
+                  description: |-
+                    Minimum consecutive failures for the probe to be considered failed after
+                    having succeeded. Defaults to 3. Minimum value is 1.
+                  default_value: 3
+                - !ruby/object:Api::Type::NestedObject
+                  name: tcpSocket
+                  description: |-
+                    TcpSocket specifies an action involving a TCP port.
+                  exactly_one_of:
+                    - tcp_socket
+                    - http_get
+                  send_empty_value: true
+                  properties:
+                    - !ruby/object:Api::Type::Integer
+                      name: port
+                      description: |-
+                        Port number to access on the container. Number must be in the range 1 to 65535.
+                - !ruby/object:Api::Type::NestedObject
+                  name: httpGet
+                  description: |-
+                    HttpGet specifies the http request to perform.
+                  exactly_one_of:
+                    - tcp_socket
```

Comment:
This needs to be the entire path to this field like:
template.0.spec.0.containers.0.startup_probe.0.tcp_socket

Reply:
done